repo/raja-perf/application.py

```diff
@@ -18,9 +18,14 @@ class RajaPerf(ExecutableApplication):
             'mpi','network-point-to-point','network-latency-bound',
             'c++','raja','sycl']
 
-    executable('run', 'raja-perf.exe --size {size} --variants {RAJAPerf_variant} --tunings {RAJAPerf_tuning} --outfile {experiment_name}', use_mpi=True)
+    executable('run', 'raja-perf.exe --size {size}', use_mpi=True)
 
-    workload('suite', executables=['run'])
```

Comment:
This should be already set here https://github.com/LLNL/benchpark/blob/b54de946fc0a5202e0c01a6007a438cbdd0c232c/systems/llnl-elcapitan/system.py#L681C76-L681C92

Reply:
[677a6f2](https://github.com/LLNL/benchpark/pull/829/commits/677a6f277cfded73aeb61859056a50e332bcd6ec)